packages/cfd/src/Helpers/constants.ts

```diff
@@ -1,8 +1,53 @@
 import { OSDetect } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { TCFDsPlatformType } from 'Components/props.types';
+
+const platformsText = (platform: TCFDsPlatformType) => {
+    switch (platform) {
+        case 'derivez':
+            return 'EZ';
+        case 'dxtrade':
+            return 'X';
+        default:
+            return '';
+    }
+};
+
+const platformsIcons = (platform: TCFDsPlatformType) => {
+    switch (platform) {
+        case 'derivez':
+            return 'DerivEz';
+        case 'dxtrade':
+            return 'Dxtrade';
+        default:
+            return '';
+    }
+};
+
+const mobileDownloadLink = (platform: TCFDsPlatformType, type: 'ios' | 'android' | 'huawei') => {
+    switch (platform) {
+        case 'dxtrade':
+            return getPlatformDXTradeDownloadLink(type);
+        case 'derivez':
+            return getPlatformDerivEZDownloadLink(type);
+        default:
+            return getPlatformDXTradeDownloadLink(type);
+    }
+};
```

Comment:
Combine the same logic by:
```suggestion
    switch (platform) {
        case 'derivez':
            return getPlatformDerivEZDownloadLink(type);
        case 'dxtrade':
        default:
            return getPlatformDXTradeDownloadLink(type);
    }
};
```